macrel/AMP_features.py

```diff
@@ -147,8 +78,8 @@ def main(args):
 
     ifile = args[1]
     ofile = args[2]
-    features = features(ifile)
```

Comment:
Why this change?

Reply:
just refactoring the variable name to reduce the chances of system confusion. In my computer when I call the code as a separate script, it returns that the variable was used before the assignment, which is pretty much solved by this small change.